jena-fuseki2/jena-fuseki-fulljar/pom.xml

```diff
@@ -38,6 +38,12 @@
       <groupId>org.apache.jena</groupId>
       <artifactId>jena-fuseki-webapp</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.logging.log4j</groupId>
+          <artifactId>log4j-web</artifactId>
```

Comment:
Exclude `org.apache.logging.log4j:log4j-web` -- it isn't necessary running as a single app in Jetty because there isn't a stop.unload issue. But it does contain another Log4j2Plugins.dat and this was the problem - shaing picked one, not merged them.

The general fix - the 3rd party shade transformer - didn't work reliably so the simple fix is to exclude the artifact in `jena-fuseki-fulljar`. The transformer is also very version sensitive.

The emergence fix script put in the log4j-core Log4j2Plugins.dat and does not include the log4j-web additions -- so evidence that the log4j-web isn't needed.